packages/documentation-site/patternfly-docs/content/developer-resources/dark-theme-handbook.md

```diff
@@ -0,0 +1,82 @@
+---
+id: Dark theme handbook
+title: PatternFly dark theme handbook
+section: developer-resources
+---
+
+## Enabling dark theme
+
+Dark theme styles ship with PatternFly CSS by default. To enable dark theme, add the class  `pf-v5-theme-dark`  to the `<html>` tag of your application. This class can be added dynamically to toggle dark theme on and off in your application.
+
+Dark theme can also be applied based on the browser’s `prefers-color-scheme` media query. The recommended way to apply dark theme based on this media query is to use Javascript(JS) to detect the media query and apply the `pf-theme-dark` class via the JS code. This allows a product to simply manipulate a single class to turn dark theme on or off based on user preference. Adam Argyle wrote [a great article](https://web.dev/building-a-theme-switch-component/#javascript) on using JS to manage dark theme preference, using the `prefers-color-scheme` media query alongside a manual button to toggle dark theme on or off. 
+
+## What’s changed?
+The following are some notable changes and differences between light and dark themes. 
+
+**Black color palette**
+
+Black color palette updated and a new lighter gray was added.
+
+| Variable(s) affected | New or updated color values|
```

Comment:
```suggestion
| Variables affected | New or updated color values|
```
nit - if these tables are final can we just use "variable" or "variables" as appropriate?